docs/product/sentry-basics/concepts/tracing/trace-view.mdx

```diff
@@ -4,47 +4,124 @@ sidebar_order: 30
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view -> Error"
+  ></iframe>
+</div>
+
+<br />
+### Helpful Tips
+
+Because debugging often involves multiple people or teams, we've made it easy to draw attention to specific areas of the trace and share a link that shows what you've highlighted with your colleagues. To do this, click on the row you'd like to draw attention to and share the resulting URL. Your colleague will see exactly what you want them to.
+
+Whatever you highlight will also be saved if you navigate away from the page and will still be there when you use your browser's back and forward buttons.
+
+If you're doing a comparison and want an easy way to go back and forth between highlighted areas of interest in your trace, you can pin the tabs. When a tab is pinned, the view will be persistent throughout your session.
+
+<br />
+<div
+  style={{
+    height: "0px",
+    paddingBottom: "calc(66.66% + 41px)",
+    position: "relative",
+    width: "100%",
+  }}
+>
+  <iframe
+    src="https://demo.arcade.software/yP3PFytK0ckfetjJTe52?embed&show_copy_link=true"
+    title="Sentry"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    allow="clipboard-write"
+    style={{
+      colorScheme: "light",
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view - Tabbing"
+  ></iframe>
+</div>
+<br />
+Sometimes, duration gaps between entries can be small, but critical to your debugging
+effort. That's why the trace view enables you to zoom into certain areas of the trace,
+either by double clicking on row elements, (which zooms into the row), or by using
+your mouse wheel.
+<br />
+<div
+  style={{
+    height: "0px",
+    paddingBottom: "calc(66.66% + 41px)",
+    position: "relative",
+    width: "100%",
+  }}
+>
+  <iframe
+    src="https://demo.arcade.software/U8JbMjFJR1KRiP4G4j4E?embed&show_copy_link=true"
+    title="Trace Details — Performance — sentry — Sentry"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    allow="clipboard-write"
+    style={{
+      colorScheme: "light",
+      height: "100%",
+      left: "0px",
+      position: "absolute",
+      top: "0px",
+      width: "100%",
+    }}
+    title="Trace view - Zooming"
+  ></iframe>
+</div>
+<br />
+The trace view has other features like search, which makes it easy to visualize matching
+spans that you can then iterate over using the up or down arrows. You can also tailor
+the layout to your preference.
+
+We're going to continue adding features that enable you to perform efficient investigation into whatever issues you might be facing. If you have feedback or requests, please share it with us. We'd love to hear how we can make your life easier and your users happier.
```

Comment:
Would be great to add a link to give feedback here if we're inviting users to do so

Reply:
Where should we link @vivianyentran ?